test/cql-pytest/test_describe.py

```diff
@@ -447,6 +447,24 @@ def test_desc_index(cql, test_keyspace):
         
         assert f"CREATE INDEX named_index ON {tbl}{maybe_space}(c)" in c_desc
 
+def test_desc_index_on_collections(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "a int primary key, b frozen<set<int>>, c map<int, int>, d list<int>, e set<int>") as tbl:
+        indexes = [
+            ("idx_b",           f"CREATE INDEX idx_b ON {tbl}(full(b))"),
+            ("idx_c_entries",   f"CREATE INDEX idx_c_entries ON {tbl}(entries(c))"),
+            ("idx_c_keys",      f"CREATE INDEX idx_c_keys ON {tbl}(keys(c))"),
+            ("idx_c_values",    f"CREATE INDEX idx_c_values ON {tbl}(values(c))"),
+            ("idx_d",           f"CREATE INDEX idx_d ON {tbl}(d)"),    # TODO: this index will fail the assert becuase its target is "values(d)"
+            ("idx_e",           f"CREATE INDEX idx_e ON {tbl}(e)"),    # TODO: this index will fail the assert becuase its target is "keys(e)"
```

Comment:
Nit: ditto

Reply:
By the way @tchaikov, is there a reason why we don't run codespell on test files? I can see that the `test` directory is on the skip list in `.github/workflows/codespell.yaml`.

Reply:
because i didn't have enough bandwidth to address all the issues found in `test` directory. there are still 52 of them:
```console
$ codespell test -L ans,datas,fo,ser,ue,crate,nd,reenable,strat,stap,te,raison | wc -l
52
```

once we fix all of them, we should be able to remove `test` from the skip list.

Reply:
i took this opportunity to fix some of them, see #19412